v2/go.mod

```diff
@@ -82,3 +82,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 )
+
+replace github.com/leaanthony/go-webview2 v0.0.0-20211022194343-1e4c8d4226f3 => ../go-webview2
```

Comment:
Will be removed as soon as the leaanthony/go-webview2#2 has been merged.

Reply:
@stffabi Added a couple of qs to the other PR. Thanks 👍

Reply:
@leaanthony What do you mean by 'qs'? I couldn't find any comments on the other PR.

Reply:
My bad @stffabi - the review was sat in draft 🤦